ui/config/data/general_config.js

```diff
@@ -7,7 +7,7 @@ UserConfig.registerOptions('general', {
       name: {
         en: 'Cactbot user directory',
         de: 'Cactbot Benutzerverzeichnis',
-        fr: 'Répertoire utilisateur Cacbot',
+        fr: 'Répertoire de l\'utilisateur de Cactbot',
```

Comment:
_Répertoire utilisateur_ is far better and more used in softwares.

Reply:
ok. Je peux garder de cactbot ?

Reply:
Oui \o